maro/rl_v3/policy_trainer/abs_trainer.py

```diff
@@ -149,3 +205,22 @@ def set_policy_state_dict(self, policy_state_dict: Dict[str, object]) -> None:
         for policy_name, policy_state in policy_state_dict.items():
             assert policy_name in self._policy_dict
             self._policy_dict[policy_name].set_policy_state(policy_state)
+
+    @abstractmethod
+    def _batch_grad_worker(
```

Comment:
`_batch_grad_worker` sounds like a worker prcess/node. 
How about name it as `_get_batch_loss`, and rename `_get_batch_loss` to `get_batch_loss`, since grad_worker will call it as a public method.

Reply:
Renamed to `atomic_get_batch_grad` to explicitly state its functionality: calculating an batch's gradients, in the atomic manner.